modules/face/include/opencv2/face/facemark.hpp

```diff
@@ -0,0 +1,113 @@
+#ifndef __OPENCV_FACEMARK_HPP__
+#define __OPENCV_FACEMARK_HPP__
+
+#include <iostream>
+#include <vector>
+#include <string>
+
+#include "opencv2/objdetect.hpp"
+#include "opencv2/objdetect/objdetect_c.h"
+#include "opencv2/imgproc/types_c.h"
```

Comment:
Why do you need legacy C-API? (especially in public headers - these includes should be removed)